sda_auth/models.py

```diff
@@ -45,8 +47,11 @@ def _load_jwt_private_key():
 
     def generate_jwt_token(self):
         """Generate a jwt token for a user."""
+        expiration_time = datetime.now() + timedelta(days=7)
```

Comment:
```suggestion
        expiration_time = (datetime.now() + timedelta(days=7)).strftime('%s')
```
Time format should be in epoch.

Reply:
This is handled by the jwt library, so we don't need to do it.